core/controllers/base.py

```diff
@@ -295,8 +299,78 @@ def dispatch(self):
                 self.handle_exception(e, self.app.debug)
                 return
 
+        handlers_has_passed_schema_validation = True
+        try:
+            self.validate_args_schema()
+        except self.InvalidInputException as e:
+            self.handle_exception(e, self.app.debug)
+            handlers_has_passed_schema_validation = False
+        except NotImplementedError as e:
```

Comment:
Put a TODO above this (and link it to an issue) to say "remove this clause once all handlers have had validation implemented".

Reply:
Sorry I am not sure can you explain a little bit, as because these clauses are not dependent on incrementally adding schema validation. Imagine a case when all the handlers has schemas but any developer forgets to add schema for a new handler in that case he will also face NotImplementedError, which is handled by this clause, please correct me if I am wrong at any point.

Reply:
Such errors as you describe should be caught by the backend tests, right? They should not be excepted here in the main code.

Reply:
Got your point, thank you for the explanation.

Reply:
Done